mltsp/featurize.py

```diff
@@ -226,9 +226,9 @@ def featurize_time_series(times, values, errors=None, features_to_use=[],
 
     if labels is None:
         if isinstance(times, (list, tuple)):
-            labels = np.arange(len(times)).astype('str')
```

Comment:
To be clear: right now you can pass in whatever labels you like, or if you're featurizing files then the labels will be filenames. This just changes the default names from '0', '1', ... to 0, 1, ... because I got tired of seeing '0', '1', '10', '100', ... in my output.